source/docs/contributing/frc-docs/style-guide.rst

```diff
@@ -71,21 +71,27 @@ Use title case for headings.
 
 Lists should have a new line in between each indent level. The highest indent should have ``0`` indentation, and subsequent sublists should have an indentation starting at the first character of the previous indentation.
 
-.. code-block:: ReST
-
-   - Block one
-   - Block two
-   - Block three
-
-     - Sub 1
-     - Sub 2
-
-   - Block four
+```ReST
+- Block one
+- Block two
+- Block three
+  - Sub 1
+  - Sub 2
+- Block four
+```
 
 ## Code blocks
 
 All code blocks should have a language specified.
 
+````md
```

Comment:
It doesn't look like this is working I don't see a code block

Reply:
<img width="436" alt="image" src="https://github.com/user-attachments/assets/19577fee-f8ec-4975-9259-fb183ae9ab3b">

seems to be fine?